oauth.ts

```diff
@@ -60,6 +60,7 @@ function getAuthorizationCode(parentWindow: BrowserWindow, toClearAuthState: boo
     });
 
     authWindow.webContents.on('will-navigate', (event, newUrl) => {
+      event.preventDefault();
       if (newUrl.startsWith(CALLBACK_URL)) {
         onCallback(newUrl);
```

Comment:
Can add the `event.preventDefault()` at the else state condition instead? 

We can also open the accessed url at the user's browser using `shell.openExternal(newUrl);`. The documentation for it is here https://www.electronjs.org/docs/api/shell#shellopenexternalurl-options